packages/compass-indexes/src/components/drop-column/drop-column.jsx

```diff
@@ -52,15 +55,15 @@ class DropColumn extends PureComponent {
    */
   render() {
     return (
-      <td className={classnames(styles['drop-column'])}>
+      <td className={containerStyles}>
         {this.isDroppable() ? (
-          <button
-            className="drop-btn btn btn-default btn-sm"
-            type="button"
+          <IconButton
+            aria-label="Delete Index"
             onClick={this.clickDropHandler.bind(this)}
+            data-testId={`drop-index-button-${this.props.indexName}`}
```

Comment:
```suggestion
            data-testid={`drop-index-button-${this.props.indexName}`}
```

Reply:
Should we consider adding something like this? https://github.com/testing-library/eslint-plugin-testing-library/blob/main/docs/rules/consistent-data-testid.md

Reply:
CC: @gribnoysup

Reply:
created a follow up: https://jira.mongodb.org/browse/COMPASS-6012